File: datasource/src/main/java/cn/covid/bean/CovidBean.java
package cn.covid.bean;

import lombok.Data;


/**
 * Author：
 * Date：2022/5/3116:33
 * Desc: 用来封装各省市疫情数据的JavaBean
 */
@Data
public class CovidBean {
    private String provinceName;            // 省份
    private String provinceShortName;       // 省份名缩写
    private String cityName;                // 城市名称
    private Integer currentConfirmedCount;  // 当前确诊人数
    private Integer confirmedCount;         // 累计确诊人数
    private Integer suspectedCount;         // 疑似病例数
    private Integer curedCount;             // 治愈人数
    private Integer deadCount;              // 死亡人数
    private Integer locationId;             // 位置id
    private Integer pid;                    // 位置id
    private String statisticsData;          // 每一天的统计数据
    private String cities;                  // 下属城市
    private String datetime;                // 爬取数据时间

}

File: process/src/main/scala/cn/covid/bean/CovidBean.scala
package cn.covid.bean

/**
 * Author：
 * Date：2022/6/1016:06
 * Desc:
 */
case class CovidBean(
    provinceName: String , // 省份
    provinceShortName: String , // 省份名缩写
    cityName: String , // 城市名称
    currentConfirmedCount: Int , // 当前确诊人数
    confirmedCount: Int , // 累计确诊人数
    suspectedCount: Int , // 疑似病例数
    curedCount: Int , // 治愈人数
    deadCount: Int , // 死亡人数
    locationId: Int , // 位置id
    pid: Int ,
    statisticsData: String , // 每一天的统计数据
    cities: String , // 下属城市
    datetime: String // 爬取数据时间
  )
